### DISCUSSION TITLE
Creare telecomando che controlli entità tende

### DISCUSSION THREAD

**Discussion by Pietrogramma** [indomus/forum#2670] [GENERALE - Discussioni generiche]
Ciao a tutti, seguendo delle guide, sono riuscito a domotizzare alcune tende a rullo, utilizzando un wemsD1 Mini che, tramite uno script EspHome, comanda un motore stepper 28BYJ-48 al quale sono collegate le tende.
Tramite Alexa e HA ora riesco ad alzare e abbassare le tende con comandi vocali o agendo sui pulsanti delle relative card su HA.
Vorrei, ora, realizzare una sorta di telecomando (sempre utilizzando un wemos) con dei pulsanti fisici che alzino o abbassino ogni singola tenda (le tende sono tre quindi avrei tre pulsanti up e tre down) ma non so come devo programmare tramite esphome il wemos che fungerà da telecomando, cosicché pigiando un pulsante fisico dica alla entità della tenda uno di alzare o abbassare la tenda e così via.
Potreste aiutarmi?
Ciao e grazie

**Comment by dante70** [comment]
ciao,ti dico come ho fatto io.
per comandare luci da diversi punti senza avere fisicamente il cablaggio ho collegato degli interrutttori e pulsanti ad un wemos configurando gli ingressi come binary sensor
questi vengono esposti su Home assistant e con automazioni faccio di conseguenza tutto.
nel tuo caso sarebbe sufficiente esporre 6 pulsanti in modo da andare a coprire le 3 tende

**Comment by Pietrogramma** [comment]
Grazie mille Dante, leggo solo ora la tua risposta, scusami se non ti ho ringraziato prima.

Perfetto, mi è chiaro come fare.

Grazie ancora e ciao
